.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: PythonCI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ["3.5", "3.8"]
+        tox-version: ["django22", "quality"]
+
+    steps:
+    - name: checkout repo
+      uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install pip
+      run: pip install -r requirements/pip.txt
+    - name: Install Dependencies
+      run: pip install -r requirements/travis.txt
+    - name: Run Tests
+      env:
+        TOXENV: python${{ matrix.python-version }}-${{ matrix.tox-version }}
+      run: tox
+    - name: run coverage
```

Comment:
this step would run with every job, it should be enough to run once, we can add a condition for this step.

Reply:
Added if condition to only run for `python3.8`.